scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -215,9 +217,7 @@ <h3 class="header">General Institute Settings</h3>
         {% for variant in variants %}
           {% set case = variant["case_obj"] %}
           <tr>
-            <td><!-- Gene -->
-              <span class="text-body">{{ variant.hgnc_symbols|join(', ') if variant.hgnc_symbols else '--'}}</span>
-            </td>
+            <td>{{ sv_genes_cell(variant) }}</td>
```

Comment:
This is nit-picking, but the same is actually used for all types of variants right now, so the name could use a generalisation to go beyond its history! 😄

Reply:
Ah no, you are right, it makes more sense if it's general. I'll change it!